go-controller/pkg/libovsdbops/acl.go

```diff
@@ -184,9 +184,10 @@ func DeleteACLsOps(nbClient libovsdbclient.Client, ops []libovsdb.Operation,
 	return modelClient.DeleteOps(ops, opModels...)
 }
 
-// DeleteACLs deletes the provided ACLs
-// portGroupNames and switchPred reminds to delete ACL references for port groups or switches
-// in case port group or switch is not completely deleted
+// DeleteACLs deletes the provided ACLs,
+// portGroupNames and switchPred reminds to delete ACL references for port groups or switches.
+// In case port group or switch is completely deleted, no need to call this function.
+// Provided ACLs should have UUID set.
```

Comment:
can we have a case where UUID is not set for deletion? I'm a bit confused on this statement...

Reply:
in general we can, but in our code now we usually get ACLs from the db by predicate, then do some additional handling and only choose some of them to be deleted. In this case using already known acl.UUID is the most optimal way.
We can always do explicit lookup before calling this function to fetch ACLs we want to delete, e.g. for deleting port group with its acls explicitly, I can search for a port group, create a list of ACLs with UUIDs referenced in this port group, and call `DeleteACLs` with that list.